### ISSUE TITLE
Upgrade from Rails 5.0.1 to Rails 5.1.6

### ISSUE THREAD

**Issue by alaxalves** [publiclab/mapknitter#601]
Follow the steps below will definitely help:
https://edgeguides.rubyonrails.org/upgrading_ruby_on_rails.html#upgrading-from-rails-5-0-to-rails-5-1
This depends on #600

**Comment by alaxalves** [comment]
Since we went straight ahead to Rails 5.2.3 as in https://github.com/publiclab/mapknitter/pull/685 this is no longer needed.
